_sass/animations.scss

```diff
@@ -2,8 +2,9 @@
 	animation-name: fadeIn;
 	-webkit-animation-name: fadeIn;
 
-	animation-duration: 1.5s;
-	-webkit-animation-duration: 1.5s;
+	animation-duration: 1.8s;
+	animation-delay: .6s;
```

Comment:
This is making the logo _appear_ for `.6s` and then flash out of sight and fade in, so it's not quite doing what you're wanting I think. If you want it to wait half a second and then start fading it, set an opacity property to `0` on the `.fadeIn` selector so it starts out hidden always.

Reply:
@kmcgillivray are you sure??? the way i'm seeing it happen is correct, in chrome anyway. page loads with no logo briefly, then it fades up.  If I add the opacity 0 to .fadeIn, it does the same, except after the animation is finished, the logo disappears